components/Grid/grid.css

```diff
@@ -0,0 +1,24 @@
+.grid {
+  display: grid;
+  gap: 1.5rem;
+}
+
+.grid.layout-2-column {
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid.layout-3-column {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.grid.layout-30-70 {
+  grid-template-columns: 1fr 3fr;
+}
+
+.grid.layout-70-30 {
+  grid-template-columns: 3fr 1fr;
+}
+
+.grid-centered {
```

Comment:
The class name `grid-centered` is giving a false impression that the grid as a container is being centered rather than the grid-items. As the properties set to it imply. It might be a good idea to change it to something more unambiguous.

Reply:
Done